src/Root.h

```diff
@@ -208,6 +210,7 @@ class cRoot
 
 	cCraftingRecipes * m_CraftingRecipes;
 	cFurnaceRecipe *   m_FurnaceRecipe;
+	std::unique_ptr<cBrewingRecipes>  m_BrewingRecipes;
```

Comment:
Remove the extra space - you're not aligning anymore, so a single space is enough.